packages/react-image/Spec.md

```diff
@@ -64,7 +64,7 @@ import { Image, ImageFit } from '@fluentui/react/lib/Image';
 ### v0 usage
 
 ```jsx
-<Image fluid src="..." alt="Example of the image component in v0" />
+<Image block src="..." alt="Example of the image component in v0" />
 ```
 
 ## Variants
```

Comment:
The table of the Proposed component props should be modified as well to reflect the changes. 
Line 87-88 changed to `shape | string | "" | "square", "rounded", "circular"`
Line 86 to be changed from `fluid` to `block`